bundles/org.openhab.binding.awattar/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,345 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="awattar"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="prices">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+
+		<label>aWATTar Hourly Price</label>
```

Comment:
The binding name is redundant and can therefore be removed.
```suggestion
		<label>Hourly Price</label>
```

Reply:
Changed

Reply:
Also found no checkstyle warning in the chenkins builds. Is there a special mvn goal for that? I use

```
mvn spotless:apply clean build
```

Reply:
This particular point doesn't generate a warning. The warning about the empty line could be found in `target/code-analysis/report.html`.